client/src/plus/deep-dives/overview/index.tsx

```diff
@@ -1,19 +1,27 @@
 import React from "react";
 
+import { Loading } from "../../../ui/atoms/loading";
 import { PageContentContainer } from "../../../ui/atoms/page-content";
 
 const App = React.lazy(() => import("./app"));
 
-export function Overview() {
-  const pageTitle = "MDN Plus Deep Dives";
+export function Overview({ pageTitle }: { pageTitle?: string }) {
+  const defaultTitle = "Modern CSS in the Real World";
```

Comment:
I'm not super confident about this. It's definitely a weakness in how Yari (and create-react-app) works. 
It's important that the `title` is correct in `tools/spas.js` and the only time you can actually NOT have that prop is if you've client-side navigated to this page. Perhaps in that scenario the `defaultTitle` should be something like `Loading Deep-dives overview app...`. But we're in a phase of desperate hacks for the benefit of the market research page.